portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -18,53 +18,66 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
-
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import java.text.SimpleDateFormat;
+import java.util.Date;
 
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  ArrayList<Comment> jsonArray = new ArrayList<Comment>();
-
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("comment-input");
-    //As of right now, comments are stored in jsonArray and datastore
+    String timeStamp = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss").format(new Date());
+    String user = "Anonymous";
 
     if (text.isEmpty() == false) {
-        //When retrieval from datastore is implemented, this section of code will change
-        Comment newComment = new Comment(text);
-        jsonArray.add(newComment);
-
-        //sending comment to datastore
-        Entity commentEntity = new Entity("Comments");
+        Entity commentEntity = new Entity("commentsData");
         commentEntity.setProperty("Comment", text);
```

Comment:
I'm not sure about this, but it might be standard to keep property names lowercase (e.g. "Timestamp" --> "timestamp").